localizer.py

```
@@ -11,7 +11,11 @@ class Localizer:
     def __init__(self) -> None:
         self.translations: dict[str, list[dict[str, object]]] = {}
 
+    async def load_translations_async(self, locale: str) -> list[dict[str, object]]:
+        return await run_blocking(self._load_translations_sync, locale)
```

Flake8 (7.3.0)

[error] undefined name 'run_blocking'

(F821)